flask_rebar/validation.py

```diff
@@ -62,6 +63,16 @@ def require_output_fields(self, data):
                     raise ValidationError(messages.required_field_missing(field_name))
                 elif field.allow_none is False and data[field_name] is None:
                     raise ValidationError(messages.required_field_empty(field_name))
+                elif field.validate:
+                    try:
+                        field.validate(data[field_name])
+                    except ValidationError as e:
```

Comment:
Added this exception handling to include the field name in the error message, not sure if that's necessary.